Makefile

```diff
@@ -250,4 +250,4 @@ test-shellcheck:
 .PHONY: test-yaml
 test-yaml:
 	@which yamllint >/dev/null 2>&1 || echo "Command 'yamllint' not found, can not execute YAML syntax checks"
-	yamllint --strict $$(git ls-files "*.yml" "*.yaml" | grep -v ^dbicdh)
+	yamllint --strict $$((git ls-files2 "*.yml" "*.yaml" 2>/dev/null || find -name '*.y*ml') | grep -v ^dbicdh)
```

Comment:
What's `git ls-files2` meant to do? Is this a typo? Or is it a local alias?

Reply:
Perhaps it's worth adding a comment pointing out what this line achieves. Why is the `|| find` used here?

Reply:
Good catch. I used "ls-files2" as temporary test. Added a comment for explanation.